### DISCUSSION TITLE
Disabling E-mail input field

### DISCUSSION THREAD

**Discussion by jeonyunsoo** [pretix/pretix#3213] [Q&A]
My organization is trying to use pretix and collect only phone numbers instead of using email to contact our customers
I`ve already setup an api that send text messages to all attendees when registration is complete but I cant find any way to disable the email input at registration. 
There seems to be a checkbox that can choose whether pretix asks for email but I cant disable it.
Any ideas how I can disable the email input field?

**Comment by raphaelm** [comment]
It's not possible in pretix to disable the email field, sorry, pretix always assumes being able to send an email
